### ISSUE TITLE
fishtape parsing depends on whitespace in a manner incompatible with `fish_indent`

### ISSUE THREAD

**Issue by braham-snyder** [jorgebucaran/fishtape#27]
```
~ ⋊> cat test_fishtape.fish
test "tautology"
    1 = 1
end

~ ⋊> fishtape test_fishtape.fish
TAP version 13
ok 1 tautology

1..1
# tests 1
# pass  1

# ok

~ ⋊> fish_indent -w test_fishtape.fish

~ ⋊> cat test_fishtape.fish
test "tautology"
1 = 1
end

~ ⋊> fishtape test_fishtape.fish
TAP version 13
not ok 1 tautology1
  ---
    operator: fail
    expected: '= or !='
    received: '= 1 '
  ...

1..1
# tests 1
# pass  0
# fail  1

~ master * ⧕ fish -v
fish, version 2.6.0
```

Thanks for `fishtape`!

**Comment by jbucaran** [comment]
Hi @braham-snyder! Can you walk me through the problem? and did fish_indent go through any changes recently I don't know maybe? 🤔

**Comment by jorgebucaran** [comment]
@braham-snyder This is fixed in 2.0! 🎉  

See the changelog:

- https://github.com/jorgebucaran/fishtape/releases/tag/2.0.0
